app/components/service/service.html

```diff
@@ -66,15 +75,35 @@
                 </div>
               </td>
             </tr>
-            <tr ng-if="service.Env">
-              <td>Env</td>
+            <tr ng-if="service.EnvironmentVariables">
+              <td>Environment Variables</td>
               <td>
-                <table class="table table-bordered table-condensed">
-                  <tr ng-repeat="var in service.Env">
-                    <td>{{ var|key: '=' }}</td>
-                    <td>{{ var|value: '=' }}</td>
-                  </tr>
-                </table>
+                <div class="form-group">
+                  <div class="col-sm-11">
+                    <span class="label label-default interactive" ng-click="addEnvironmentVariable(service)">
+                      <i class="fa fa-plus-circle" aria-hidden="true"></i> environment variable
+                    </span>
+                  </div>
+                  <!-- environment-variable-input-list -->
+                  <div class="col-sm-11 form-inline" style="margin-top: 10px;">
+                    <div ng-repeat="var in service.EnvironmentVariables" style="margin-top: 2px;">
+                      <div class="input-group col-sm-5 input-group-sm">
+                        <span class="input-group-addon">name</span>
+                        <input type="text" class="form-control" ng-model="var.key" ng-disabled="var.added" placeholder="e.g. FOO">
```

Comment:
To be consistent with the service creation view, the placeholder for a var name should be `com.example.foo`